tools/themer.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+"""Compiles themes for wasp-os"""
+
+from argparse import ArgumentParser, RawTextHelpFormatter
+from importlib import import_module
+from typing import Tuple
+
+class DefaultTheme():
+    """This class represents the theme. Import this file and extend the Theme class, only changing the variables.
+Export the resulting class as 'Theme'.
+"""
```

Comment:
Indentation should match the start of the docstring (there is magic in the docstring parsing to allow sensibble indentation). I think it should also be

~~~
    """A single summary sentance.

    More detailed description spanning
    multiple lines and indented to match the opening
    quote.
    """"
~~~